src/types/License.ts

```diff
@@ -9,16 +11,99 @@ export interface LicenseRef {
 }
 
 export interface License extends LicenseRef {
+    billing_company_name: string;
+    billing_contact: string;
+    billing_contact_email: string;
+    /**
+     * Billing cycle in months. (e.g. 1, 3, 6, 12, 24 or 36)
+     * If billing cycle info is not manually set
+     * it will be null (no data about billing cycle).
+     */
+    billing_cycle: number | null;
+    /**
+     * Date when subscription will be automatically canceled.
+     */
+    cancel_at: string | null;
+    /**
+     * Date when subscription cancellation was requested.
+     */
+    canceled_at: string | null;
+    company_address: string;
+    company_address_city: string;
+    company_address_country: string;
+    company_address_zip: string;
     created_at: string;
+    /**
+     * When there are no subscriptions yet, the currency is null (we do not set a default value).
+     */
+    currency: 'eur' | 'usd' | 'gbp' | null;
```

Comment:
This looks like something that could be extracted into an enum :)

Reply:
Will do. Thanks!

Reply:
434d8bd7faf939eb99a05999d972f4690a433cf9